qa/tasks/ceph_manager.py

```diff
@@ -1671,14 +1674,14 @@ def expand_pool(self, pool_name, by, max_pgs):
             self.pools[pool_name] = new_pg_num
             return True
 
-    def set_pool_pgpnum(self, pool_name):
+    def set_pool_pgpnum(self, pool_name, force):
         """
         Set pgpnum property of pool_name pool.
         """
         with self.lock:
             assert isinstance(pool_name, basestring)
             assert pool_name in self.pools
-            if self.get_num_creating() > 0:
```

Comment:
"fix_pgp_num" is also a thrashosd action. i guess the thrash tests are supposed to be quick. but it can be skipped if the pool has pgs being created. in the case of "healthy" task, it will wait anyway, so no need to block in a loop for waiting the num_creating to be 0. so i think the current fix is good enough. what do you think?